theme/assets/js/ui/dialog.js

```diff
@@ -28,7 +29,8 @@ document.addEventListener('alpine:init', () => {
     });
 
     Alpine.store('dialog', {
-        currentDialogConfig: {},
+        primaryDialogConfig: {},
+        secondaryDialogConfig: {},
```

Comment:
I see we access these by an index number, are they intended to be objects or arrays?